rules.d/70-uaccess.rules.in

```diff
@@ -99,4 +99,8 @@ SUBSYSTEM=="hidraw", ENV{ID_AV_PRODUCTION_CONTROLLER}=="1", TAG+="uaccess"
 # This also allows accessing HID devices with the libusb backend of hidapi.
 SUBSYSTEM=="usb", ENV{ID_AV_PRODUCTION_CONTROLLER}=="1", TAG+="uaccess"
 
+# Trezor HW wallets
```

Comment:
And then drop this comment to make this stuff generic

Reply:
Changed this to `Hardware wallets`